net/hev-socks5-server/files/hev-socks5-server.config

```diff
@@ -0,0 +1,5 @@
+config hev-socks5-server 'enabled'
```

Comment:
Since it's a pretty simple configuration, you can merge `enabled` option into `config` section directly.